src/app/games/game/game-progress/game-chart/game-chart.component.ts

```diff
@@ -34,14 +34,19 @@ export class GameChartComponent implements OnChanges {
 
   ngOnChanges(): void {
     // don't do anything until all inputs are provided
-    if (typeof this.matching !== "string" || typeof this.nonmatching !== "string" || typeof this.metadata !== "object") {
+    if (/* typeof this.line_names !== "object" || */ typeof this.datasets !== "object" || typeof this.metadata !== "object") {
```

Comment:
Any reason why we aren't checking `line_names` anymore? If we aren't using it, please remove it instead of just commenting it out